core/src/main/java/io/questdb/griffin/engine/groupby/vect/NSumDoubleVectorAggregateFunction.java

```diff
@@ -70,18 +72,18 @@ public void aggregate(long address, long addressSize, int columnSizeHint, int wo
             // Neumaier compensated summation
             final double x = Vect.sumDoubleNeumaier(address, addressSize / Double.BYTES);
             if (x == x) {
-                final int offset = workerId * Misc.CACHE_LINE_SIZE;
-                final double sum = this.sum[offset];
+                final int sumOffset = workerId * SUM_PADDING;
+                final double sum = this.sum[sumOffset];
                 final double t = sum + x;
-                double c = this.sum[offset + 1];
+                double c = this.sum[sumOffset + 1];
                 if (Math.abs(sum) >= x) {
                     c += (sum - t) + x;
                 } else {
                     c += (x - t) + sum;
                 }
-                this.sum[offset] = t; // sum = t
-                this.sum[offset + 1] = c;
-                this.count[offset]++;
```

Comment:
I'm just guessing because I don't know the surrounding code but wouldn't assigning sumOffset and sumOffset+1 introduce false sharing if padding is decreased to 64 bytes  ?

Reply:
It should not in theory, each writer has one cache line to play with. They should be able to write anywhere in that, as long as they don't write into each other lines.

Reply:
Fixed in 3facf05b48c20e191ba7699f21ffd54672cedc55